tutorials/fundamentals/trim.md

```diff
@@ -0,0 +1,27 @@
+To trim leading and trailing characters from a string in JavaScript, you should use the `replace()` function.
+`trim()` is only useful for removing whitespace characters which is why you should use `replace()`.
+`replace()` takes two arguments:
+- a regular expression denoting what to take out
+- a string denoting what to put in
+
+```javascript
+let example = 'aaaaaaaaaaaaaaaaaaaaBBBaaaaaaaaaaaaaaaaaaaa';
+example.replace(/^a+/, 'f').replace(/a+$/, 'g'); // BBB
+```
+
+You could also do this using a single `replace()` function if you wanted the same character before and after the middle part.
```

Comment:
Let's get rid of this sentence and the below example, because `example.replace(/a+/,'')` doesn't "trim". It removes all instances of "a", not just leading/trailing "a".